pkgs/by-name/th/thrift-ls/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "thrift-ls";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "joyme123";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-Qib/xbTQiR0VpHsCOt5HGkbytaQ0GSCqaHVYFNPYGVs=";
+  };
+
+  vendorHash = "sha256-YoZ2dku84065Ygh9XU6dOwmCkuwX0r8a0Oo8c1HPsS4=";
+
+  postInstall = ''
+    mv $out/bin/thrift-ls $out/bin/thriftls
+  '';
+
+  meta = with lib; {
+    description = "Language server for Thrift";
```

```suggestion
    description = "Thrift Language Servert";
```
This is more personal opinion than anything, however this follows the description on the source repo.